tools/verify/e5-t18d-local-fixtures.py

```
@@ -0,0 +1,162 @@
+def crashed(pid):
+    os.kill(int(pid), signal.SIGKILL)
+
+def failed(child, reason, timeout=15):
+    child.wait(timeout=timeout)
+    assert child.returncode == 0
+    assert text('desktop.failed').strip() == reason, text('desktop.log')
+    assert not text('desktop.ready') and not text('weston.pid')
+    assert 'event=fallback reason=' + reason in text('desktop.log')
+
+run('cc', '-Wall', '-Wextra', '-Werror', str(ROOT / 'tools/verify/fixtures/e5-t18d-process.c'), '-o', '/usr/bin/weston')
+shutil.copy('/usr/bin/weston', '/tmp/seatd')
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/seatd"

(S108)